src/api/app/views/webui/distributions/new.html.haml

```diff
@@ -0,0 +1,39 @@
+- @pagetitle = "Add distribution repository to #{@project}"
+
+
+.card.mb-3
+  = render partial: 'webui/project/tabs', locals: { project: @project }
+  .card-body
+    %h3 Add Repositories to #{@project}
+    .row
+      - @distributions.each do |vendor, distributions|
+        .col-md-12
+          .pb-4
+            %h6.pb-2
+              - if resolve_asset_path("distributions/#{vendor.downcase}.svg")
+                = image_tag("distributions/#{vendor.downcase}.svg")
+              #{vendor} distributions
+              %span.badge.badge-primary
```

Comment:
I don't know about this... Is it really needed?

Reply:
Needed? No

Reply:
I should have phrased my previous comment more like `I don't think knowing the total of distributions is really useful`